packages/purpose-process/src/resources/templates/documents/riskAnalysisTemplate.html

```diff
@@ -48,12 +56,24 @@ <h1>Analisi del rischio</h1>
       </div>
 
       <div class="item">
-        <div class="label">Erogatore</div>
+        <div class="label">Ente erogatore</div>
         <div class="value">{{producerText}}</div>
       </div>
 
+      {{#if producerDelegationId}}
+      <div class="item">
+        <div class="label">Ente delegato all’erogazione</div>
+        <div class="value">
+          {{producerDelegateName}}{{#if producerDelegateIpaCode}} (codice IPA:
+          {{producerDelegateIpaCode}}){{/if}}
+        </div>
+      </div>
+      {{/if}}
+
+      <br />
+
       <div class="item">
-        <div class="label">Fruitore</div>
+        <div class="label">Ente fruitore</div>
         <div class="value">{{consumerText}}</div>
```

Comment:
Replace `consumerText` with the same approach used for producer delegate: name + IpaCode if present